mpl_point_clicker/_clicker.py

```diff
@@ -66,35 +74,51 @@ class in *classes*
             colors = [None] * len(self._classes)
         if markers is None:
             markers = ["o"] * len(self._classes)
+        if disable_legend is None:
+            disable_legend = False
+        if disable_legend and len(self._classes) != 1:
+            disable_legend = False
```

Comment:
```suggestion
```

I understand the urge here - but we shouldn't override the user input here. They may be changing the current class programmatically or with their own gui elements. Also with `disable_legend` default to False we don't need to check for `None`